syntax_and_semantics/literals/array.md

```diff
@@ -13,33 +13,34 @@ Arrays are typically created with an array literal denoted by square brackets (`
 The array's generic type argument `T` is inferred from the types of the elements inside the literal. When all elements of the array have the same type, `T` equals to that. Otherwise it will be a union of all element types.
 
 ```crystal
-[1, 2, 3]          # => Array(Int32)
-[1, "hello", 'x']  # => Array(Int32 | String | Char)
+[1, 2, 3]         # => Array(Int32)
```

Comment:
Why?

Reply:
When you would pass this code to `crystal tool format` then it would format it like this but there are a lot more examples which aren't formatted. Maybe I should do this in another PR.